### PR TITLE
Fix image tagging for bottlerocket bootstrap image

### PR DESCRIPTION
**Pull request by abhay-krishna**
This is so it gets the `v<release branch>-<release number>-latest` tag.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Common.mk [hunk 1]
```diff
@@ -132,7 +132,7 @@ IMAGE_USERADD_USER_NAME?=
 
 # Branch builds should look at the current branch latest image for cache as well as main branch latest for cache to cover the cases
 # where its the first build from a new release branch
-IMAGE_IMPORT_CACHE?=type=registry,ref=$(LATEST_IMAGE) type=registry,ref=$(IMAGE:$(lastword $(subst :, ,$(IMAGE)))=latest)
+IMAGE_IMPORT_CACHE?=type=registry,ref=$(LATEST_IMAGE) type=registry,ref=$(subst $(LATEST),latest,$(LATEST_IMAGE))
```
Comment (abhay-krishna) [inline Common.mk:135]: This is to fix entire tag being replaced with `latest` for images like kind, bottlerocket-bootstrap, cloud-provider vsphere etc, which contain other release-branch specific elements in the tag.

File: projects/aws/bottlerocket-bootstrap/Makefile [hunk 2]
```diff
@@ -13,7 +13,7 @@ SOURCE_PATTERNS=.
 REPO_SUBPATH=..
 
 GIT_TAG=v$(RELEASE_BRANCH)-$(shell yq e ".$(RELEASE_BRANCH).number" $(BASE_DIRECTORY)/EKSD_LATEST_RELEASES)
-
+LATEST_TAG=$(GIT_TAG)-$(LATEST)
```
Comment (jaxesn) [inline projects/aws/bottlerocket-bootstrap/Makefile:16]: This should already be set from the common.mk... I'll take a closer look here

Comment (jaxesn) [reply projects/aws/bottlerocket-bootstrap/Makefile:16]: Yeah remove these changes and instead move https://github.com/aws/eks-anywhere-build-tooling/blob/main/Common.mk#L35 to  https://github.com/aws/eks-anywhere-build-tooling/blob/main/Common.mk#L100

File: Common.mk (hunk 1 above)
Comment (jaxesn) [reply Common.mk:135]: This looks right, good catch!  Lets backport this to release06 when it merges.

**Review by jaxesn — APPROVED** [review]
/lgtm

**Comment by abhay-krishna** [comment]
/override bottlerocket-bootstrap-tooling-presubmit
/override cert-manager-tooling-presubmit
/override cloud-provider-vsphere-tooling-presubmit
/override cluster-api-provider-aws-tooling-presubmit
/override cluster-api-provider-tinkerbell-tooling-presubmit
/override cluster-api-provider-vsphere-tooling-presubmit
/override cluster-api-tooling-presubmit
/override cri-tools-tooling-presubmit
/override eks-anywhere-cli-tools-tooling-presubmit
/override eks-anywhere-diagnostic-collector-tooling-presubmit
/override eks-anywhere-test-image-tooling-presubmit
/override etcdadm-bootstrap-provider-tooling-presubmit
/override etcdadm-controller-tooling-presubmit
/override etcdadm-tooling-presubmit
/override flux-tooling-presubmit
/override govmomi-tooling-presubmit
/override hegel-tooling-presubmit
/override helm-controller-tooling-presubmit
/override imagebuilder-presubmit
/override kind-tooling-presubmit
/override kube-rbac-proxy-tooling-presubmit
/override kube-vip-tooling-presubmit
/override kustomize-controller-tooling-presubmit
/override local-path-provisioner-tooling-presubmit
/override notification-controller-tooling-presubmit
/override source-controller-tooling-presubmit
/override troubleshoot-tooling-presubmit
/override vsphere-csi-driver-tooling-presubmit
/override cfssl-tooling-presubmit

Overriding since these presubmits have already run successfully, and are now just reconciling with merges to the base branch.

**Pull request merged**